Bottom: e261c290f12b21d7670b532a2af281339a9c54db
Top:    33263bf95f37406ccc72ec88d108cc62859b0b8b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-23 19:12:05 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index c5159ba..da2e3b3 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2752,7 +2752,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
